### ISSUE TITLE
net-im/telegram-desktop: Bump required version of media-libs/libwebp

### ISSUE THREAD

**Issue by Anonymous1157** [rindeal/gentoo-overlay#123]
For quite a few versions now, I have been experiencing a crash with the following console message when Telegram attempts to load the full-resolution version of some stickers:
```
telegram-desktop: /var/tmp/portage/media-libs/libwebp-0.4.0/work/libwebp-0.4.0/src/dec/vp8l.c:619: GetHtreeGroupForPos: Assertion `meta_index < hdr->num_htree_groups_' failed.
Aborted
```
The crash is caused by only a handful of stickers, and I know of exactly four in the nearly thousands I have seen. I am not able to determine the exact problem, but upgrading to `=media-libs/libwebp-0.4.2` (Currently marked testing in Portage) seems to resolve the issue with Telegram.

If anyone else would like to do testing, the "Bazinga" and "This" stickers in this pack have caused the crash for me:

https://telegram.me/addstickers/ToxicAnswers

**Comment by rindeal** [comment]
Thank you for the report. I suppose you're using version 0.9.56, aren't you?

**Comment by Anonymous1157** [comment]
Oops, forgot to mention that! I'm actually running 0.10.1-r1, but I'm 100% certain this has been happening for me for many versions.

**Comment by rindeal** [comment]
Hmm, the fix is a no-brainer for v0.10, but v0.9.56 will have to go back to unstable :disappointed: 

Also please report it sooner next time, it could have been fixed already.
